client/src/js/components/bhFiltersApplied.js

```diff
@@ -0,0 +1,14 @@
+angular.module('bhima.components')
+.component('bhFiltersApplied', {
+  controller: FiltersAppliedController,
```

Comment:
I do not think that you need this controller, since it doesn't do anything.  What happens if you remove it?